src/templates/Page.js

```diff
@@ -17,6 +17,8 @@ export const Page = (props) => {
 
   const content = markdown(htmlAst);
 
+  const twoDigitNumber = number < 9 ? '0' + number : number;
```

Comment:
LOOK @tofumatt I MADE A TERNARY OR WHATEVER. Please send prizes and/or statues in honour of my Javascriptyness immediately.